model/enum.go

```diff
@@ -0,0 +1,158 @@
+package model
+
+import (
+	"log"
+	"regexp"
+)
+
+type EntityType int
+
+const (
+	Undefined EntityType = iota
+	Dataset
+	DataService
+	Concept
+	InformationModel
+	PublicService
+	Event
+)
+
+func ParseEntityType(str *string) EntityType {
+	if str == nil {
+		return Undefined
+	}
+
+	dataset_match, err := regexp.Match("(?i)Dataset", []byte(*str))
+	if err != nil {
+		log.Println("error parsing dataset EntityType")
+		return Undefined
+	}
+
+	dataService_match, err := regexp.Match("(?i)DataService", []byte(*str))
+	if err != nil {
+		log.Println("error parsing DataService EntityType")
+		return Undefined
+	}
+
+	concept_match, err := regexp.Match("(?i)Concept", []byte(*str))
+	if err != nil {
+		log.Println("error parsing Concept EntityType")
+		return Undefined
+	}
+
+	informationModel_match, err := regexp.Match("(?i)InformationModel", []byte(*str))
+	if err != nil {
+		log.Println("error parsing InformationModel EntityType")
+		return Undefined
+	}
+
+	publicService_match, err := regexp.Match("(?i)PublicService", []byte(*str))
+	if err != nil {
+		log.Println("error parsing PublicService EntityType")
+		return Undefined
+	}
+
+	event_match, err := regexp.Match("(?i)Event", []byte(*str))
+	if err != nil {
+		log.Println("error parsing Event EntityType")
+		return Undefined
+	}
+
+	if dataset_match {
+		return Dataset
+	}
+	if dataService_match {
+		return DataService
+	}
+	if concept_match {
+		return Concept
+	}
+	if informationModel_match {
+		return InformationModel
+	}
+	if publicService_match {
+		return PublicService
+	}
+	if event_match {
+		return Event
+	}
+	return Undefined
+}
+
+func (e EntityType) ToPath() *string {
+	var path string
+	switch e {
+	case Dataset:
+		path = "datasets/"
+	case DataService:
+		path = "dataservices/"
+	case Concept:
+		path = "concepts/"
+	case InformationModel:
+		path = "informationmodels/"
+	case PublicService:
+		path = "public-services-and-events/"
```

Comment:
```suggestion
		path = "public-services/"
```
Eller er det meningen at dette skal være begge?

Reply:
ofc, ignore this